src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    private String name;
+    private boolean done;
+
+    public Task(String name) {
+        this.name = name;
+        this.done = false;
+    }
+
+    public void mark() {
+        this.done = true;
+    }
+
+    public void unmark() {
+        this.done = false;
+    }
+
+    @Override
+    public String toString() {
+        if (this.done) {
+            return "[X] " + this.name;
+        } else {
+            return "[ ] " + this.name;
+        }
+    }
```

Comment:
Alternative suggestion: maintain another method to generate either the [X] or [ ], since the this.name is redundant with both the conditions